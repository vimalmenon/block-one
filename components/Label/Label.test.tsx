import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { Label } from './'

describe('Label Component', () => {
  test('Label: Snapshot testing', () => {
    const wrapper = shallow(<Label>Test Data</Label>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
