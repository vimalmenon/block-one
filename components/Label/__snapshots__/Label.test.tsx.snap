// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Component Label: Snapshot testing 1`] = `
<label>
  Test Data
</label>
`;
